Src/Fido2/Objects/AttestedCredentialData.cs

```diff
@@ -16,6 +16,7 @@ public sealed class AttestedCredentialData
     /// <see cref="https://www.w3.org/TR/webauthn/#attested-credential-data"/>
     /// </summary>
     private readonly int _minLength = Marshal.SizeOf(typeof(Guid)) + sizeof(ushort) + sizeof(byte) + sizeof(byte);
```

Comment:
change to const int _minLength = 12; // guid (8) + short (2) + byte (1) + byte (1) ?

Reply:
Eh, I kind of like this one the way it is, matches the sizes mentioned in the spec.  Guid is 16 bytes, not 8.

Reply:
Ugh -- yes, Guid = 16. Mostly wanted to kill the call and make this a const, but given my own mistake above, maybe best as is! :)

Reply:
I am just going to go with a `readonly int` value of 20.  This isn't spec level check, it's a sanity check added so we can tell when the conformance test that sends a value that is way too small we can tell that is specifically the test we passed and not some other problem.